diagnostic.md

```diff
@@ -16,13 +16,14 @@ Graces-MacBook-Pro:try grace$
 
 What is the full path to the current directory?
 
-_Replace this text with your answer._
+/Users/Benjamski/wdi/fundamentals-diagnostic/diagnostic.md
```

Comment:
It was supposed to be the path in the question, not your current path, but thats fine